src/main/java/org/gridsuite/sensitivityanalysis/server/SensitivityAnalysisController.java

```diff
@@ -212,6 +215,28 @@ public ResponseEntity<String> getDefaultProvider() {
         return ResponseEntity.ok().body(service.getDefaultProvider());
     }
 
+    @PostMapping(value = "/results/{resultUuid}/csv")
+    @Operation(summary = "export sensitivity results as csv file")
+    @ApiResponses(@ApiResponse(responseCode = "200", description = "Sensitivity results successfully exported as csv file"))
+    public ResponseEntity<byte[]> exportSensitivityResultsAsCsv(@Parameter(description = "Result UUID") @PathVariable("resultUuid") UUID resultUuid,
+                                                                @RequestBody SensitivityAnalysisCsvFileInfos sensitivityAnalysisCsvFileInfos) {
+        try {
```

Comment:
as comment in the service file, e do not need try catch

Reply:
Removed